jfsd/main.py

```diff
@@ -154,6 +154,14 @@ def main(Nsteps: int,
                  boundary_flag,thermal_test_flag,alpha_friction,ho_friction)
     return trajectory, stresslet_history, velocities, test_result
     
+def overlap_monitor(displacements_vector_matrix,N):
```

Comment:
Functions should generally be with a verb, so I might suggest `check_overlap` in this case, because that is what it does?

Reply:
I see you also have a `check_overlaps` already. Perhaps you can rename that one to `find_overlaps` to distinguish better?

Reply:
Modified as suggested.